apps/frontend/services/api/kobo/useForm.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  DisasterType,
+  DroughtDto,
+  FloodDto,
+  IncidentDto,
+} from '@wfp-dmp/interfaces';
+import path from 'path';
+import useSWR from 'swr';
+
+import { ApiRoutes } from 'services/api/apiRoutes';
+import { apiClient } from 'services/api/client';
+
+export const useForm = (formDisasterType: DisasterType, formId: string) => {
+  console.log('in hook');
+  const { data } = useSWR(
+    [ApiRoutes.form, formDisasterType, formId],
+    async (relativePath, disasterType, id) => {
+      const url = path.join(relativePath, disasterType, id);
```

Comment:
use a combination of new URL with path.join to accomplish this, i think this is a bad practice https://javascript.plainenglish.io/how-to-safely-concatenate-url-with-node-js-f6527b623d5

Reply:
My base url is a relative path here. It is not working with URL